docs/Jenkins & Docker.txt

```diff
@@ -0,0 +1,24 @@
+Jenkins and Docker are installed on the remote server so there is no need to have them on your local machine.
+To be able to operate them you have to receive your accounts. Once they are created for you ->
+Login to -web UI     https://bonnee.eu/ci/ 
+         -CLI        ssh accountName@bonnee.eu   
```

Comment:
-CLI        ssh accountName@bonnee.eu    -> - SSH: ssh accountName@bonnee.eu

* currently the VPS is owned and maintained by @petr-vanis-cgi  . If you need SSH access to the VPS he can create a user for you.